publish.build.gradle

```diff
@@ -1,20 +1,20 @@
 apply plugin: 'maven-publish'
 apply plugin: 'signing'
 
-dokka {
-    outputFormat = "html"
-    outputDirectory = "$buildDir/javadoc"
```

Comment:
The documentation discusses being able to set this but it appears it doesn't exist. I'm operating under the assumption that when using the `dokkaJavadoc` block, it's implicitly javadoc, as opposed to `dokkaHtml` and other default configs.